### ISSUE TITLE
Add File and Path Validation in plot() Function of piechart.py

### ISSUE THREAD

**Issue by Shreya7tripathy** [this-is-yaash/sneaky-package#31]
Hi, I would like to suggest adding file and path validation to the plot() function in piechart.py. Currently, the function assumes that the specified source file exists and that the output path is valid. However, this can lead to issues if:

1.The file in the "stash" directory does not exist.
2.There are problems with read/write permissions.
3.The necessary directories for writing output do not exist.
Adding validation for file existence, directory creation, and basic error handling would make the function more robust and prevent unexpected errors. These improvements are widely regarded as best practices for file handling if I am not wrong.

I’m a complete beginner, so if this issue is correct, could you please assign it to me @this-is-yaash ? I'd love to work on it! 😊

**Comment by this-is-yaash** [comment]
That's a good idea!  
But keep in mind, this package is designed to integrate files into another computer in a stealthy way.

So, make sure to display error messages minimally. For example, if the file `auth` doesn't exist in the `stash` directory, you can show a message like 'check for syntax errors.' The message should not alert anyone monitoring the computer to what you're doing.

Ensure the error messages are minimal and non-alerting. If you can manage that, leave a comment again, and I'll assign this issue to you!

**Comment by this-is-yaash** [comment]
1. Basically, you'll check whether the file exists in the directory or not.  
2. If it does, perform the respective operation.  
3. If not, print a custom error message. That's it!

**Comment by Shreya7tripathy** [comment]
Yes I can definitely give it a try. I am determined to complete this. If I face any issues I will contact you as it is my first time, please do guide me. So, please do assign me this.

**Comment by this-is-yaash** [comment]
@Shreya7tripathy Yeah, good to hear that!
Feel free to reach out to me, when you need assistance.

Now I'll be assigning this issue to you.

**Comment by Shreya7tripathy** [comment]
Thank you !!

**Comment by jindalpriyanshu101** [comment]
Completed with #36
